deployment/docker-compose/pretix/pretix/etc/pretix.cfg

```diff
@@ -0,0 +1,28 @@
+[pretix]
+instance_name=My pretix installation
+url=http://127.0.0.1/
+currency=EUR
+datadir=/data
+
+[database]
+backend=mysql
+name=pretix
+user=pretix
+password=pretix
+host=db
+
+[redis]
+location=redis://redis/0
+sessions=true
+
+[celery]
+backend=redis://redis/1
+broker=redis://redis/2
+
+[uwsgi]
+http =  0.0.0.0:8000
+master = True
+processes = 4
+threads = 2
+module = pretix.wsgi
+cron = -30 -1 -1 -1 -1 cd /pretix/src && python3 -m pretix runperiodic
```

Comment:
This file should also include the following section:
```ini
[django]
secret=ooShah3UpeiCohngoop0phaeghequah0aor5ohpa3xeephie9E
debug=off
```
where secret is the django encryption secret.